include/ceed/sycl.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+#ifndef _ceed_sycl_h
+#define _ceed_sycl_h
+
+#include <ceed/ceed.h>
```

Comment:
This file appears to be empty. Is it needed?

Reply:
File will be deleted in subsequent commit. It was intended to be kept as a placeholder to match other backends.